src/transformers/generation/configuration_utils.py

```diff
@@ -188,7 +188,15 @@ class GenerationConfig(PushToHubMixin):
         guidance_scale (`float`, *optional*):
             The guidance scale for classifier free guidance (CFG). CFG is enabled by setting `guidance_scale > 1`.
             Higher guidance scale encourages the model to generate samples that are more closely linked to the input
-            prompt, usually at the expense of poorer quality.
+            prompt, usually at the expense of poorer quality. If `negative_prompt` is unset, this will set it to the
+            last input token.
+        negative_prompt (`torch.LongTensor` of shape `(batch_size, sequence_length)`, *optional*):
+            The negative prompt to use for CFG. Will throw an error if set but `cfg_scale` <= 1 or None. The batch size
+            must match the input batch size.
+        cfg_rescale (float, *optional*):
+            Rescale CFG for improved quality. Expected values in range [0,1]. Reducing its value smoothens the CFG
+            effects and recovers some lost quality. Lower values allows for higher guidance scales. No effect unless
+            `guidance_scale` is set too. Default: 1.0.
```

Comment:
```suggestion
            `guidance_scale` is set too.
```
We want to keep unset configuration parameters as `None`, as it is much easier to check whether they are set or not :)